src/uct/api/v2/uct_v2.h

```diff
@@ -198,6 +209,47 @@ typedef struct uct_md_mem_dereg_params {
 } uct_md_mem_dereg_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_iface_is_reachable_v2.
+ */
+typedef struct uct_iface_is_reachable_params {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref uct_iface_is_reachable_field_mask_t. Fields not specified in this
+     * mask will be ignored. Provides ABI compatibility with respect to adding
+     * new fields.
+     */
+    uint64_t                     field_mask;
+
+    /**
+     * Device address to check reachability to.
+     * This field is not relevant if iface_attr.dev_addr_len == 0.
+     */
+    const uct_device_addr_t      *dev_addr;
+
+    /**
+     * Interface address to check reachability to.
+     * This field is not relevant if iface_attr.iface_addr_len == 0.
+     */
+    const uct_iface_addr_t       *iface_addr;
+
+    /**
+     * User-provided pointer to a string buffer.
+     * The function @ref uct_iface_is_reachable_v2 fills this buffer with the
+     * null-terminated information string why the remote address is not
+     * reachable if the return value is 0.
+     */
+    char                         *info_str;
+
+    /**
+     * The length of the @a info_str is provided in bytes.
+     * This value must be specified in conjunction with @a info_str.
+     */
+    size_t                        info_str_len;
```

Comment:
maybe rename to info_str_length (to have full word)?

Reply:
fixed